src/client/src/routes/StyleguideRoute/styled/Paragraph.tsx

```diff
@@ -0,0 +1,17 @@
+import { styled, Styled } from 'rt-theme'
+
+import { mapMarginPaddingProps, MarginPaddingProps } from './mapMarginPaddingProps'
+import { mapTextProps, TextProps } from './Text'
+
+export interface ParagraphProps extends TextProps, MarginPaddingProps {}
```

Comment:
`type ParagraphProps = TextProps & MarginPaddingProps`

Reply:
But, is there any reason to not describe it as `interface`? For example, can a `type` be extended by another `interface`?